src/common/substrait/src/df_substrait.rs

```diff
@@ -40,13 +39,11 @@ impl SubstraitPlan for DFLogicalSubstraitConvertor {
     async fn decode<B: Buf + Send>(
         &self,
         message: B,
-        catalog_manager: CatalogManagerRef,
+        catalog_list: Arc<dyn CatalogList>,
```

Comment:
nit: do we need to define an alias `CatalogListRef`?

Reply:
> nit: do we need to define an alias `CatalogListRef`?

This `CatalogList` is datafusion's trait so I didn't alias it.